hub/providers/Proposal/createProposal.ts

```diff
@@ -30,9 +31,15 @@ import {
 } from 'actions/createProposal';
 import { tryGetNftRegistrar } from 'VoteStakeRegistry/sdk/api';
 
+import {
```

Comment:
`hubs` folder is not maintained so no changes in this file should matter; sorry about that